app/services/submission_service.rb

```diff
@@ -311,5 +311,20 @@ def update_submission_info(artwork, submission)
       )
       submission.save!
     end
+
+    def add_user_to_submission(submission, gravity_user_id, access_token)
+      if gravity_user_id.present?
+        user = User.find_or_create_by(gravity_user_id: gravity_user_id)
```

Comment:
**thought (non-blocking):** It's so confusing to have two sets of user records to manage. I wish this were simpler...